packages/collector/test/tracing/database/redis/connect-via/default.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * (c) Copyright IBM Corp. 2024
+ */
+
+'use strict';
+
+module.exports = async function (redis, log) {
+  const client = redis.createClient({ url: `redis://${process.env.REDIS}` });
```

Comment:
It would be better to add comments on how to run the redis tests locally.

Reply:
Already discussed in previous comment.